codebase2/codebase-sqlite/U/Codebase/Sqlite/Queries.hs

```diff
@@ -1939,40 +1932,63 @@ termNamesWithinNamespace bhId mayNamespace = do
         WHERE
           root_branch_hash_id = :bhId
           AND namespace GLOB :namespaceGlob
+
+        UNION ALL
```

Comment:
The goal is to remove all uses of `termNamesWithinNamespace` eventually in favour of the precise pretty-printer, but we still use it for a few things in the meantime.

For backwards compatibility we have it return every name within the project of whatever is being viewed/searched and one layer of dependencies.